otherfile/font_wizard.py

```diff
@@ -102,8 +102,11 @@ def is_ascii(s):
             print(Error2)
             sys.exit()
     font_dic = dict(zip(Letters, font_data))
+    font_split_list = list(map(lambda x: len(x.split("\n")), font_dic.values()))
+    font_height = font_split_list[0]
```

Comment:
Shouldn't it be `font_height = len(font_split_list[0])`?

Reply:
No, but I didn't choose a proper name for variable names.
Fixed in https://github.com/sepandhaghighi/art/pull/185/commits/9f01cc4c984e141bfa20e94f9688b6c253e7f851